### DISCUSSION TITLE
Request to Reconsider Cancellation of My Application

### DISCUSSION THREAD

**Discussion by ajaynegi45** [community/community#50049] [GitHub Education]
Dear GitHub Education Department,

I hope this email finds you well. I am writing to request your kind reconsideration of the cancellation of my student developer pack application.

I applied for the student developer pack, but I received a notification stating that my application has been cancelled. I understand that the cancellation was due to the unavailability of a student email and ID card from my college. Unfortunately, my college has not yet provided me with these requirements, as the process is still ongoing.

However, I have just completed my first-semester exams and have obtained my admit card for the same. I believe that this should serve as a valid proof of my student status.

As a dedicated and passionate learner, I am looking forward to utilizing the benefits of the GitHub Student Developer Pack to enhance my skills and knowledge. Therefore, I humbly request you to reconsider my application and grant me access to the student developer pack.

Thank you for your time and consideration. I look forward to hearing from you soon.

**Comment by samus-aran** [comment]
Hi @ajaynegi45 👋 

We're happy you're here. You are able to re-apply if you have more appropriate evidence of your academic status. Here are a few tips on applying.

1. Ensure you select the most appropriate **evidence type** in the dropdown. The order of options indicates the preference of evidence with Academic Email being at the top. If you don't have an Academic email or Student ID, do try and acquire evidence that is positioned higher rather than lower in that option list.
2. Ensure your image (photo or upload) is legible. If you are unable to read its text on the image itself then the verification process may not be able to either. If you are taking a photo of a screen try to have it as large a resolution as possible - take a photo of the evidence on a monitor rather than a phone perhaps.

Hope this helps and let us know how you get on 👍
